README.md

```diff
@@ -1,25 +1,26 @@
 # geodev-hackerlabs
 
-A place to learn how to build geo apps with the ArcGIS Platform.
+> A place to learn how to build geo apps with the ArcGIS Platform.
 
 ![Steps](./geodev-hackerlabs-apps.png)
 
 ## Getting Started
 
-Follow these steps to prepare for the labs.
+Sign up for an [ArcGIS Developer subscription](https://developers.arcgis.com/en/sign-up/).
 
-1. Sign up for an [ArcGIS Developer subscription](https://developers.arcgis.com/en/sign-up/).
-2. Login to [ArcGIS Online](http://arcgis.com).
- * Search for `Geodev Hackerlabs Share` > `Search Groups`.
- * Uncheck `Only search in my account/organization`.
- * Join the group.
-3. Sign up for a GitHub account if you don't have one.
+The subscription allows you to:
+* Upload/Import data
+* Create new services
+* Store maps and other content online
+* Perform spatial analysis
+
+> If you already have an Organizational account, that will work too.  You **cannot** use a [free public account](https://geonet.esri.com/groups/geodev/blog/2014/10/15/help-i-cant-get-into-developersarcgiscom).
```

Comment:
added a note at the bottom explaining that Org accounts are okay, but free public accounts are not.